R/aggregateSolute.R

```diff
@@ -74,7 +74,7 @@
 #'   predictions (FALSE)?
 #' @param attach.units logical. If true, units will be attached as an attribute 
 #'   of the second column of the returned data.frame.
-#' @param complete.threshold numeric decimal value below which an \code{agg.by}
+#' @param complete.threshold numeric number of observations below which an \code{agg.by}
```

Comment:
good. sorry about flip-flopping on this earlier - i do think number of observations will give enough control and more clarity to users